### PR TITLE
Refactor/catch

### PR DESCRIPTION
**Pull request by irmakerdem**
What does this PR do?
- Update catch error message 🤡
- Updates location of fetch calls
- Removes old code

Where should your reviewer start?
- Line 1 in apiCalls.js
- Line 5 in scripts.js

What (if any) features are you implementing?
- N/A

What (if anything) did you refactor?
- Removed pseudocode and console logs

Were there any issues that arose?
- N/A

Is there anything that you need from your teammate?
- N/A

Any other comments, questions, or concerns?
- N/A

### CODE REVIEW

File: src/apiCalls.js [hunk 1]
```diff
@@ -1,5 +1,20 @@
...
 
+  let endpoints = apis.map((url) => {
+    return fetch(url)
+    .then(response => response.json())
+    .catch(error => alert("🤡 OOOPSIES! 🤨 There was an error. Please try again. 🤡 "));
+  })
+  
+  const allData = Promise.all(endpoints).then((value) => {
+    return value
+  });
```
Comment (catlynbowles) [inline src/apiCalls.js:17]: I like how all the resolved data is only entering scripts now 🦹🏼‍♀️

File: src/scripts.js [hunk 2]
```diff
@@ -53,25 +54,13 @@ function getAllSleepData(data) {
...
-    getAllHydrationData(value[1].hydrationData);
-    getAllSleepData(value[3].sleepData);
-  });
+function displayResolvedData() {
+  fetchAllData()
+  .then((allData) => {
+    getAllUserData(allData[0].userData);
+    getAllHydrationData(allData[1].hydrationData);
+    getAllSleepData(allData[3].sleepData);
+  })
```
Comment (ASands17) [inline src/scripts.js:63]: I love how descriptive this function name is! 🌼

File: src/apiCalls.js [hunk 3]
```diff
@@ -1,5 +1,20 @@
...
+    'https://fitlit-api.herokuapp.com/api/v1/users',
+    'https://fitlit-api.herokuapp.com/api/v1/hydration',
+    'https://fitlit-api.herokuapp.com/api/v1/activity',
+    'https://fitlit-api.herokuapp.com/api/v1/sleep'
+  ];
 
+  let endpoints = apis.map((url) => {
+    return fetch(url)
+    .then(response => response.json())
+    .catch(error => alert("🤡 OOOPSIES! 🤨 There was an error. Please try again. 🤡 "));
```
Comment (catlynbowles) [inline src/apiCalls.js:12]: love the catch displaying to user 🦤

**Review by catlynbowles — APPROVED** [review]

File: src/apiCalls.js (hunk 3 above)
Comment (ASands17) [inline src/apiCalls.js:12]: Good work alerting the user if there is an issue! We might try doing this on the dom in a later iteration. 🐙

**Review by ASands17 — APPROVED** [review]
LOOKING GOOD TEAM! ✨

**Pull request merged**